src/card/info-status/info-status-card.html

```diff
@@ -5,18 +5,18 @@
          class="info-img"/>
     <span class="info-icon {{$ctrl.status.iconClass}}"></span>
   </div>
-  <div>
+  <div class="info-content-container">
     <h2 class="card-pf-title" ng-if="$ctrl.status.title">
       <a href="{{$ctrl.status.href}}" ng-if="$ctrl.status.href">
         <span class="">{{$ctrl.status.title}}</span>
       </a>
       <span ng-if="!$ctrl.status.href">
             <span class="">{{$ctrl.status.title}}</span>
-          </span>
+      </span>
     </h2>
-    <p ng-if="$ctrl.shouldShowHtmlContent" ng-bind-html="$ctrl.trustAsHtml(item)"
-       ng-repeat="item in $ctrl.status.info track by $index"></p>
-    <p ng-if="!$ctrl.shouldShowHtmlContent" ng-bind="item"
-       ng-repeat="item in $ctrl.status.info track by $index"></p>
+    <div ng-if="$ctrl.shouldShowHtmlContent" class="info-content" ng-bind-html="$ctrl.trustAsHtml(item)"
```

Comment:
nit. but I think 'showHtmlContent' would be enough of a description :-)

Reply:
@dtaylor113 I like your nits. Name follows other names around though, looking for buyin to rid ourselves of `should` in the whole of this work? `pfinfostatuscard`

Reply:
Probably not in this PR if it's being used elsewhere.  -thanks